seed/landing/views.py

```diff
@@ -8,7 +8,6 @@
 import json
 import logging
 
-import botocore.exceptions
```

Comment:
not all seed builds have botocore 😬

Reply:
oof, bad assumption on my end. Thanks for updating the message.